CANTroller2/src/qpid.h

```diff
@@ -486,6 +487,10 @@ class ThrottleControl {  // Soren - To allow creative control of PID targets in
         runstate = driving;
     }
     void update (void) {  // this should be called to update idle and throttle target values before throttle-related control loop outputs are calculated
+        // update engine temp if it's ready
+        if (engine_sensor) {
+            float engine_temp_f = engine_sensor->get_temperature();

```

Comment:
engine_temp_f is deleted after this if-statement, probably not what you want

Reply:
Good catch! I want the private member variable updated